app/controllers/CheckYourAnswersForAgentsController.scala

```diff
@@ -92,7 +92,7 @@ class CheckYourAnswersForAgentsController @Inject() (
                       )
                     )
                 }
-              case _                           =>
+              case _                                         =>
```

Comment:
As `AuthUserType` is a sealed type, we should avoid using wildcard matches
Ideally this would cause compilation to fail

Reply:
The pattern match is on `Option[AuthUserType]` though which does not benefit from the exhaustive match compilation check.

Reply:
```suggestion
              case Some(AuthUserType.IndividualTrader)                     =>
                logger.warn(
                  "Invalid journey: User navigated to check your answers for agents with IndividualTrader user type"
                )
                Future.successful(Redirect(routes.UnauthorisedController.onPageLoad))
              case None                                      =>
```